ocean/sub_ice_shelf_2D/5km/z_level/config_forward.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0"?>
+<config case="forward">
+	<add_link source="../init_step2/ocean.nc" dest="init.nc"/>
+	<add_link source="../init_step2/graph.info" dest="graph.info"/>
+
+	<add_executable source="model" dest="ocean_model"/>
+
+	<namelist name="namelist.ocean" mode="forward">
+		<template file="template_forward.xml" path_base="script_resolution_dir"/>
+		<template file="global_stats.xml" path_base="script_core_dir" path="templates/analysis_members"/>
+		<option name="config_land_ice_flux_mode">'pressure_only'</option>
+		<option name="config_disable_thick_all_tend">.true.</option>
+		<option name="config_disable_thick_hadv">.true.</option>
+		<option name="config_disable_thick_vadv">.true.</option>
+		<option name="config_disable_thick_sflux">.true.</option>
+		<option name="config_disable_vel_all_tend">.true.</option>
+		<option name="config_disable_vel_coriolis">.true.</option>
+		<option name="config_disable_vel_pgrad">.true.</option>
+		<option name="config_disable_vel_hmix">.true.</option>
+		<option name="config_disable_vel_surface_stress">.true.</option>
+		<option name="config_disable_vel_explicit_bottom_drag">.true.</option>
+		<option name="config_disable_vel_vmix">.true.</option>
+		<option name="config_disable_vel_vadv">.true.</option>
+		<option name="config_disable_tr_all_tend">.true.</option>
+		<option name="config_disable_tr_adv">.true.</option>
+		<option name="config_disable_tr_hmix">.true.</option>
+		<option name="config_disable_tr_vmix">.true.</option>
+		<option name="config_disable_tr_sflux">.true.</option>
+		<option name="config_disable_tr_nonlocalflux">.true.</option>
+		<option name="config_disable_vel_vadv">.true.</option>
+		<option name="config_disable_redi_k33">.true.</option>
```

Comment:
@cbegeman, in all these tests, do we want to leave these as disabled by default?

Reply:
No, I think we should enable all (set to `.false.`).

Reply:
Okay, which means just taking all of these out.  No need to set them to the default.